File: Training Exercises/Back From The Park/sample.pl
@a = split' ', <STDIN>;
@b = split' ', <STDIN>;

$x = $a[0];
$y = $a[1];
$x1 = $b[0];
$y1 = $b[1];

if ($x < $x1){
	print("Right\n");
}
else{
	print("Left\n");
}

File: Training Exercises/Back From The Park/sample.php
<?php
    $stdin = fopen('php://stdin', 'r');
	$stdout = fopen('php://stdout', 'w');
	
	$line_1 = fgets($stdin);
	$first = explode(' ', $line_1);
	$line_2 = fgets($stdin);
	$second = explode(' ', $line_2);

	if ($second[0] > $first[0])
		fwrite($stdout, "Right");
	else fwrite($stdout, "Left");
